### PR TITLE
add example for reduce ops

### PR DESCRIPTION
**Pull request by qjing666**
add example for reduce op and check the correction of results when dim = []

### CODE REVIEW

File: api_examples/paddle/fluid/layers/reduce_op.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import paddle.fluid as fluid
```
Comment (guru4elephant) [inline api_examples/paddle/fluid/layers/reduce_op.py]: You should follow example template:)

**Review by guru4elephant — COMMENTED** [review]
please follow others' example template

File: api_examples/paddle/fluid/layers/reduce_op.py (hunk 1 above)
Comment (qjing666) [reply api_examples/paddle/fluid/layers/reduce_op.py]: Add copyright, api description and comments

**Review by guru4elephant — APPROVED** [review]
LGTM

**Pull request merged**